docs/reading_files.rst

```diff
@@ -268,3 +268,34 @@ You only need these details if you think caching may be causing problems.
 
 JSON was chosen as it can be easily inspected manually, and it's reasonably
 efficient to load the entire file.
+
+File Reading Issues
+-------------------
+
+Files at European XFEL storage migrate over time from GPFS to PNFS. The data 
+on PNFS usually always available for reading. But sometimes, reading may require
+staging from the tape to dCache. If there is staging queue, the operation can take
```

Comment:
```suggestion
staging from the tape to disk. If there is a staging queue, the operation can take
```

We tend to talk about the whole of `/pnfs` as dCache, both the disks and tape storage. I'm not sure if that's technically accurate, but it works pretty well for our purposes, so I'd rather not use it differently here.